### DISCUSSION TITLE
How to save a routing congestion heat map report?

### DISCUSSION THREAD

**Discussion by Leonardo-SA** [The-OpenROAD-Project/OpenROAD#5737] [Q&A] [answered]
Hi,

I can visualize the Routing Congestion Heat Map in the GUI, but I'm unsure how to save it in a format other than just an image. Could you please guide me on how to save a routing congestion heat map report?

Thank you

**Comment by maliberty** [comment, marked as answer]
See https://openroad.readthedocs.io/en/latest/main/src/gui/README.html#dump-heatmap-to-file
